### PR TITLE
Added the error 404 page and hooked it up with React Router to the in…

### PR DESCRIPTION
**Pull request by Wavist**
…dex.js

Signed-off-by: Wavist <sommy.okafor@yahoo.com>

### CODE REVIEW

File: src/Error404.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import './Error404.css'
```
Comment (hypermystic) [inline src/Error404.js:2]: Some linting errors only semicolon missing

File: src/App.js [hunk 2]
```diff
@@ -236,7 +236,7 @@ function App() {
       );
       ReactDOM.render(fileBtn, document.getElementById("fileBtn"));
     } else {
-      history.push('/?key='+keys[0]);
+      history.push('/keys/?key='+keys[0]);
```
Comment (hypermystic) [inline src/App.js:239]: /?key will also work if we wrap the routing in the index.js in Switch

Comment (Wavist) [reply src/App.js:239]: Yes.. i completely forgot to wrap it. Will send a new PR

Comment (Wavist) [reply src/App.js:239]: Done

File: src/index.js [hunk 3]
```diff
@@ -5,15 +5,17 @@ import App from './App';
...
-import { Route, BrowserRouter as Router } from 'react-router-dom'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
 
 const routing = (
   <Router>
+    <Switch>
     <Route exact path="/signup" component={Signup} />
     <Route exact path="/profile" component={Profile} />
     <Route exact path="/" component={App} />
     <Route path="/keys/?key=:id" component={App}/>
```
Comment (hypermystic) [inline src/index.js:16]: <Route path='/?key=:id' component={App} />
should be this

File: src/App.js (hunk 2 above)
Comment (hypermystic) [reply src/App.js:239]: Should be 
history.push('/?key='+keys[0]);

**Review by hypermystic — CHANGES_REQUESTED** [review]
Please see the changes and rest looks fine to me

File: src/index.js (hunk 3 above)
Comment (hypermystic) [reply src/index.js:16]: Do the required change here also

Comment (Wavist) [reply src/index.js:16]: Okay

**Comment by hypermystic** [comment]
Thanks for the page @Wavist, we have fixed some issues related to this in #27

**Pull request merged**